### PR TITLE
Enhance eduard msgs: Part 2

### PR DESCRIPTION
**Pull request by Kichura**
1. Removes Waterfall reference from `profilekey`/`missingkey` msg as it's EOL,
2. Adds italic to texts that have brackets in them,
4. Adds commas to specific texts where it's appropriate, *(Same for columns)*
5. `Viarewind` -> `ViaRewind`,
6. ~~Syncs the proxy command text with the other text although the duplicate should be cut out if needed.~~ Removes the duplicate proxy command,
7. Places Velocity above BungeeCord in terms of proxy command.

### CODE REVIEW

File: src/main/resources/messages.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
                 "proxy",
                 "velocity"
             ],
-            "message": "Please update your proxy to the latest version to support 1.8 - latest release clients\n**BungeeCord** - https://ci.md-5.net/job/BungeeCord\n**Velocity** - https://papermc.io/downloads#Velocity"
+            "message": "Please update your proxy to the latest version to support 1.8 - latest release clients:\n**BungeeCord** - https://ci.md-5.net/job/BungeeCord\n**Velocity** - https://papermc.io/downloads#Velocity"
```
Comment (kennytv) [inline src/main/resources/messages.json:17]: Swap them here instead

File: src/main/resources/messages.json [hunk 2]
```diff
@@ -107,7 +107,7 @@
         },
         {
             "command": "proxy",
```
Comment (kennytv) [inline src/main/resources/messages.json:109]: This is a duplicate, just remove it

**Review by kennytv — CHANGES_REQUESTED** [review]

File: src/main/resources/messages.json (hunk 1 above)
Comment (Kichura) [reply src/main/resources/messages.json]: Swapped.

File: src/main/resources/messages.json (hunk 2 above)
Comment (Kichura) [reply src/main/resources/messages.json]: Removed.

**Review by kennytv — APPROVED** [review]

**Pull request merged**